### DISCUSSION TITLE
Memperkecil gap antar fluidrow shinnydashboard

### DISCUSSION THREAD

**Discussion by DeanAlexander27** [teamalgoritma/community#568] [Q&A] [answered]
Dear Ka Instruktur

bagaimana cara untuk memperkecil gap nya ya ?
![image](https://user-images.githubusercontent.com/123348110/220869696-20a25aef-ce8a-4822-aaa1-5b96b6f2f1e9.png)

berikut koding UI nya

library(shiny)
library(shinydashboard)

ui <- dashboardPage( skin = "red",
                     
                     dashboardHeader(title = "PT.A Supply Chain Division"),
                     
                     dashboardSidebar(
                       sidebarMenu(
                         menuItem("Dashboard Year on Year", tabName = "dashboard", icon = icon("dashboard")),
                         menuItem("Dashboard Category", tabName = "dashboard category", icon = icon("dashboard"),
                                  badgeLabel = "New !", badgeColor = "green"),
                         menuItem("Distribution Progress", icon = icon("th"), tabName = "widgets",
                                  badgeLabel = "New !", badgeColor = "green"),
                         menuItem("Data",icon = icon("users"),tabName = "widgets",
                                  badgeLabel = "New !", badgeColor = "green")
                       )
                     ),
                     
                     dashboardBody(
                       tags$style(HTML("
    .small-h2 {
      font-size: 16px;
    }
  ")),
  
  
  tabItems(
    tabItem(tabName = "dashboard",
            h1(strong("Supply Chain Dashboard"), align ="center"),
            h2(strong("PT.A plays a vital role as the lead distributor of motor engines, and its success is measured by key metrics such as sales, distribution, and total stock levels."), align ="center", class="small-h2"),
            br(),
            fluidRow(box(plotlyOutput(outputId = "progress2022"), width = 9),
                     column(width = 3,
                            infoBox(title = "Total Sales 2022", width = NULL,value = tail(datasetsales1$total_sales,1), icon = icon("credit-card"), fill = TRUE,color='red'),
                     infoBox(title = "Distribution Stock 2022", width = NULL,value = tail(datasetmandistnew1$total_dist,1), icon = icon("truck"),fill = TRUE,color='orange'),
                     infoBox(title = "Total Stock 2022", width = NULL,2685, icon = icon("boxes-stacked"),fill = TRUE),
                     tabBox(
                       title = "Market Information",
                       # The id lets us use input$tabset1 on the server to find the current tab
                       id = "tabset1",height = "250px", width = NULL,
                       tabPanel("Positive Info", "First tab content"),
                       tabPanel("Negative Info", "Tab content 2")
                     )))
              ,
            fluidRow(box(plotlyOutput(outputId = "progress2022.1"), width = 9))
            
              ),
            
            tabItem(tabName = "dashboard category",
                    h2("Widgets tab content")),
            tabItem(tabName = "widgets",
                    h2("Widgets tab content")),
            tabItem(tabName = "widgets",
                    h2("Widgets tab content"))
            
            
    )
                     )
                     )

**Comment by finesaaa** [comment, marked as answer]
Dear @DeanAlexander27,
Silahkan dicoba dengan memberikan margin sebagaimana jawaban dari diskusi pada referensi berikut
_You can use negative values on margin, in this case use margin-top:-2em to affect only the top margin. I prefer to use relative units, but you can use pixel instead of em._
https://stackoverflow.com/questions/45249955/how-do-i-eliminate-stubborn-white-space-between-fluidrows-in-shiny

**Comment by DeanAlexander27** [comment]
Dear Ka @Fiqey Indriati

terima kasih ka.. sudah tersolved
